explore_data/nbconverted/0_explore_input_data.py

```diff
@@ -0,0 +1,180 @@
+
+# coding: utf-8
+
+# In[1]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+
+import os
+import sys
+import glob
+import pandas as pd
+import numpy as np
+import random
+import seaborn as sns
+import umap
+from keras.models import load_model
+from sklearn.decomposition import PCA
+import pickle
+
+from plotnine import (ggplot,
+                      labs,  
+                      geom_line, 
+                      geom_point,
+                      geom_errorbar,
+                      aes, 
+                      ggsave, 
+                      theme_bw,
+                      theme,
+                      xlim,
+                      ylim,
+                      facet_wrap,
+                      scale_color_manual,
+                      guides, 
+                      guide_legend,
+                      element_blank,
+                      element_text,
+                      element_rect,
+                      element_line,
+                      coords)
+
+from ponyo import utils
+
+from numpy.random import seed
+randomState = 123
+seed(randomState)
+
+
+# In[2]:
+
+
+# Read in config variables
+base_dir = os.path.abspath(os.path.join(os.getcwd(),"../"))
+
+config_file = os.path.abspath(os.path.join(base_dir,
+                                           "config_human.tsv"))
+params = utils.read_config(config_file)
+
+
+# In[3]:
+
+
+# Load params
+local_dir = params["local_dir"]
+project_id = params['project_id']
+
+
+# In[4]:
+
+
+# Load real template experiment
+template_data_file = params['template_data_file']
+
+# Load metadata file with grouping assignments for samples
+metadata_file = os.path.join(
+    base_dir,
+    "data",
+    "metadata",
+    project_id+"_groups.tsv")
+
+
+# In[5]:
+
+
+# Read data
+template_data = pd.read_csv(
+    template_data_file,
+    header=0,
+    sep='\t',
+    index_col=0)
+
+template_data.head()
+
+
+# In[6]:
+
+
+"""
+# Try different partitions of the data
+smRNA_samples = ["SRR493961",
+                 "SRR493962",
+                 "SRR493963",
+                 "SRR493964",
+                 "SRR493965",
+                 "SRR493966",
+                 "SRR493967",
+                 "SRR493968",
+                 "SRR493969",
+                 "SRR493970",
+                 "SRR493971",
+                 "SRR493972"]
+#template_data = template_data.loc[smRNA_samples]
+template_data = template_data.drop(smRNA_samples)
+print(template_data.shape)
+template_data.head()"""
```

Comment:
Is this cell supposed to be run, or is it a debug statement? Also do you have a commented out debug statement in your commented out debug statement?

Reply:
This cell is supposed to run and was commented out while debugging.